pkg/services/rds/lastoperation.go

```diff
@@ -15,47 +14,44 @@ func (b *RDSBroker) LastOperation(instanceID string, operationData database.Oper
 
 	// Log opts
 	b.Logger.Debug(fmt.Sprintf("lastoperation rds instance opts: instanceID: %s operationData: %v", instanceID, models.ToJson(operationData)))
+	ids, err, getRealErr := SyncStatusWithReal(b, instanceID, operationData.ServiceID,
```

Comment:
'getRealErr' looks like a method name, maybe 'realErr' ?

Reply:
+1, and `db_instance` is more readable than `ids`, I prefer `db_instance, db_error, service_error:= ...`